src/ngraph/op/util/rnn_cell_base.hpp

```diff
@@ -71,10 +73,48 @@ namespace ngraph
                 /// \return     The object representing activation function.
                 ///
                 ActivationFunction get_activation_function(std::size_t idx) const;
+                ///
+                /// \brief      Creates node with element-wise add operation with numpy broadcasting.
+                ///
+                /// \param[in]  lhs   The left hand side argument node.
+                /// \param[in]  rhs   The right hand side argument node.
+                ///
+                /// \return     Node with element-wise add operation.
+                ///
+                static std::shared_ptr<Node> add(const std::shared_ptr<Node>& lhs,
+                                                 const std::shared_ptr<Node>& rhs);
+                ///
+                /// \brief      Creates node with element-wise subtract operation with numpy broadcasting.
+                ///
+                /// \param[in]  lhs   The left hand side argument node.
+                /// \param[in]  rhs   The right hand side argument node.
+                ///
+                /// \return     Node with element-wise subtract operation.
+                ///
+                static std::shared_ptr<Node> sub(const std::shared_ptr<Node>& lhs,
+                                                 const std::shared_ptr<Node>& rhs);
+                ///
+                /// \brief      Creates node with element-wise multiply operation with numpy broadcasting.
+                ///
+                /// \param[in]  lhs   The left hand side argument node.
+                /// \param[in]  rhs   The right hand side argument node.
+                ///
+                /// \return     Node with element-wise multiply operation.
+                ///
+                static std::shared_ptr<Node> mul(const std::shared_ptr<Node>& lhs,
+                                                 const std::shared_ptr<Node>& rhs);
+                ///
+                /// \brief      Creates node with element-wise clip operation with numpy broadcasting.
+                ///
+                /// \param[in]  data   The input tensor for clipping.
+                ///
+                /// \return     Node with element-wise clip operation.
+                ///
+                std::shared_ptr<Node> clip(const std::shared_ptr<Node>& data) const;
 
             private:
-                std::size_t m_hidden_size = 0.f;
-                float m_clip = 0.f;
+                const std::size_t m_hidden_size = 0.f;
+                const float m_clip = 0.f;
```

Comment:
Should this be `const` or `static const`? (Is it possible for it to be initialized differently elsewhere?)

Reply:
Sorry, maybe this default member initialization is confusing. I'll get rid of it. This should be only `const` - since it shouldn't change during op lifetime (actually we may let them change, however then we would have to ensure that the fused op subgraph is rebuild) and these are an individual object property. One `xxxCell` node instance may differ from the another. Since each `xxxCell` node shares these attributes I've decided to put them into common base class.